webapp/components/Spinner.tsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable sort-keys */
```

Comment:
Let's not disable this.

```suggestion
```

If you disable it for the Props, let's sort them

Reply:
if you disable it for the `sizeMap`, I prefer to disable only for the lines that show an error, and not for the entire file. See how we do something similar in [tailwind config](https://github.com/hemilabs/ui-monorepo/blob/3e3f63e379f03a21c5f35ab0e62e60099b259e36/webapp/tailwind.config.ts#L86) (And in line [192](https://github.com/hemilabs/ui-monorepo/blob/3e3f63e379f03a21c5f35ab0e62e60099b259e36/webapp/tailwind.config.ts#L192))